frontend.java

```diff
@@ -0,0 +1,540 @@
+		System.out.print("Command>");
+		String command = input();
+		if(command.equals("login")){
+			error("You are already logged in.");
+		}else if(command.equals("logout")){
+			front_logout();
+		}else if(command.equals("create")){
+			if(type == 2){
+				front_create();
+			}else{
+				error("Not logged into the proper account.");
+			}
+		}else if(command.equals("delete")){
+			if(type == 2){
+				front_delete();
+			}else{
+				error("Not logged into the proper account.");
+			}
+		}else if(command.equals("deposit")){
+			front_deposit();
+		}else if(command.equals("withdraw")){
+			front_withdraw();
+		}else if(command.equals("transfer")){
+			front_transfer();
+		}else{
+			error("Some error statement.");
+		}
+	}
+	public static void error(String errorMessage){
+		System.out.println(errorMessage);
+		takeCommand();
+	}
+	public static void front_create() {
+		// get the account number
+		// get the account name
+		// update the transaction summary file
+		System.out.print("Account Number:");
+		String accountNumber = frontend.input();
+		if(accountNumber(accountNumber)==false){
+			frontend.error("Error: Invalid account number.");
+		}else{
+			//validate the account number 
+			System.out.print("Account Name:");
+			String accountName = frontend.input();
+			if(accountName(accountName)==false){
+				frontend.error("Error: Invalid account name.");
+			}else{
+			//validate the account name 
+
+			
+				accountNumber = addZeros(accountNumber, 6);//adds zeros in front if size is less than 6
+
+				String str = "04_" + accountNumber + "_" + accountName; 
+				frontend.transactionFile.add(str);
+				//write.writeToSummaryFile(str);
+				System.out.println("Account created.");
+			}
+		}
+	}
+	public static void front_logout() {
+		frontend.loggedIn = 0;
+		//run main function on frontend.java
+		System.out.println("Logged out.");
+		
+		for (String s : frontend.transactionFile){
+			writeToSummaryFile(s);	
+		}
+    	frontend.login();
+	}
+	public static void front_delete() {
+		System.out.print("Please enter an account number:");
+		String accountNumber = frontend.input();
+		if(accountNumber(accountNumber)==false){
+			frontend.error("Error: Invalid account number.");
+		}else{
+			System.out.print("Please enter an acount name:");
+			String accountName = frontend.input();
+			if(accountName(accountName)==false){
+				frontend.error("Error: Invalid account name.");
+			}else{
+
+				accountNumber = addZeros(accountNumber, 6);//adds zeros in front if size is less than 6
+				
+				String str = "05_" + accountNumber + "_" + accountName;
+				frontend.transactionFile.add(str);
+				//write.writeToSummaryFile(str);
+				System.out.println("Account deleted.");
+			}
+		}
+	}
+	public static void front_deposit() {
+		//get account number
+		//get amount
+		System.out.print("Account Number:");
+		String accountNumber = frontend.input();
+		if(accountNumber(accountNumber)==false){
+			frontend.error("Error: Invalid account number.");
```

Comment:
if only frontend.error() isn't written to restart the command loop.

If all error() did was to display the error message, what you could do is to return early, which would make this function far more readable.

e.g. 
```
if (! isValidAccountNumber(accountNumber))
{
    error("Error: invalid account number.");
    return;
}
System.out.print("Account Amount:");
// etc etc
```